cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,71 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPage from '../support/pages/signUp.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
```

Comment:
You're importing a HomePageObject class but it's only used once. Consider reusing it in other tests or moving the `assertHeaderContainUsername` method to a more common class if it's not specific to the home page.

Reply:
I use 'clickNewArticle' in 'Article' testsuite and 'clickSettings' in 'Settings' testsuite